test-data/unit/check-classes.test

```diff
@@ -1665,6 +1665,49 @@ class D:
 main:4: error: Invalid signature "def (__main__.B, __main__.A) -> __main__.B"
 main:6: error: Invalid signature "def (__main__.C, builtins.str, builtins.str) -> __main__.C"
 
+[case testSetAttr]
+from typing import Union
+class A:
+    def __setattr__(self, name: str, value: Any) -> None: ...
```

Comment:
A few others:
- `__setattr__` is defined, but not a callable
- `__setattr__` is defined but takes the wrong arguments